static/js/settings_user_groups.js

```diff
@@ -37,7 +37,7 @@ exports.populate_user_groups = function () {
                 description: data.description,
             },
         }));
-        var pill_container = $('.pill-container[data-group-pills="' + data.name + '"]');
+        var pill_container = $(".pill-container[data-group-pills=" + JSON.stringify(data.name) + "]");
```

Comment:
I think we might want to use an HTML escape thing rather than JSON.stringify?

Another thought is: do we actually need names here at all, or can we just do the ID?

Reply:
`data-group-pills` attribute is always the group name, though I suspect we could we could migrate over to using IDs only. What do you mean by HTML escape thing?

Reply:
By HTML escape, I mean something like `Handlebars.Utils.escapeExpression` that is designed for escaping HTML for insertion into the DOM safely.

Reply:
@timabbott I see, I used the jQuery [escapeSelector function](https://api.jquery.com/jQuery.escapeSelector/) instead (works fine with the string ">`!@#$%^&*()_+{}":<>?∫Ω≈ç√∫˜µ≤¬åπøß∂œˆ∑ø∂˙ø˙ƒ∑á" and amended the commit, let me know if anything else needs changing!